packages/ra-no-code/src/ResourceConfiguration/ResourceConfiguration.tsx

```diff
@@ -102,7 +102,7 @@ export const ResourceConfigurationPage = () => {
                 <FormWithRedirect
                     save={save}
                     initialValues={resourceConfiguration}
-                    render={({ handleSubmitWithRedirect }) => (
+                    render={({ handleSubmit }) => (
```

Comment:
this should be documented in the upgrade guide

Reply:
You mean the difference between `handleSubmit` and `handleSubmitWithRedirect` and the fact that `handleSubmitWithRedirect` was removed? If yes, it already is